kiwi/repository/zypper.py

```diff
@@ -256,8 +268,7 @@ def import_trusted_keys(self, signing_keys):
         for key in signing_keys:
             # Including --dbpath flag is a workaround for bsc#1112357
```

Comment:
I believe we should delete this comment, shouldn't we? There is no `--dbpath` anymore.